### PR TITLE
Log on report_up and report_down

### PR DESCRIPTION
**Pull request by TheNavigat**
Built a package and ran it on one of our internal servers. Logs show up as expected.

### CODE REVIEW

File: lib/nerve/reporter/zookeeper.rb [hunk 1]
```diff
@@ -74,6 +73,8 @@ def stop()
     end
 
     def report_up()
+      log.info "nerve: adding zk node at #{@full_key}" if @full_key
```
Comment (mks-m) [inline lib/nerve/reporter/zookeeper.rb]: i think i would prefer these messages in `zk_create`, `zk_save` and `zk_delete` methods, as close to actual ZK operations as possible

Comment (mks-m) [inline lib/nerve/reporter/zookeeper.rb]: also, what is `@full_key` going to stringify into? i think reporting `@key_prefix` would be cleaner for human consumption unless you're specifically looking for some additional zk-related data.

**Review by mks-m — APPROVED** [review]
not blocking, this can ship if you ran the code and happy with how and where those log lines show up

File: lib/nerve/reporter/zookeeper.rb (hunk 1 above)
Comment (TheNavigat) [reply lib/nerve/reporter/zookeeper.rb]: `/smartstack/global/server_side_rendering.gondola-themed-ads-management-portal/10-112-59-86-uswest1astagef_0000000050` is an example of `@full_key`. I need to check an example of `key_prefix` and see how it looks like

Comment (mks-m) [reply lib/nerve/reporter/zookeeper.rb]: yeah, that looks like useful info, let's use `@full_key`

**Comment by TheNavigat** [comment]
@mks-m Requesting another review now that I have successfully tested the package and moved the logging code around a bit

**Review by Rob-Johnson — APPROVED** [review]

**Review by mks-m — APPROVED** [review]

**Review by ankit864 — APPROVED** [review]

**Pull request merged**